BasiliskII/src/slirp/udp.c

```diff
@@ -338,14 +338,10 @@ udp_attach(so)
     addr.sin_port = 0;
     addr.sin_addr.s_addr = INADDR_ANY;
     if(bind(so->s, (struct sockaddr *)&addr, sizeof(addr))<0) {
-      int lasterrno=errno;
+      int error = WSAGetLastError();
```

Comment:
See slirp.h starting at line 38.  Admittedly, I haven't tested building for other platforms.